docs/collecting-data/collecting-from-own-applications/mobile-trackers/migration-guides/migration-guide-from-version-5-x-to-6-0/index.md

```diff
@@ -0,0 +1,82 @@
+
+### Platform context entity (iOS only)
+
+To comply with Apple's Privacy Manifest rules, we have removed the automatic tracking of `totalStorage` and `availableStorage` metrics from the [platform context entity](docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/platform-and-application-context/index.md). We've also added a Privacy Manifest for the SDK.
+
+To track `totalStorage` or `availableStorage`, use the new `PlatformContextRetriever` callbacks class. The `PlatformContextRetriever` is available for both iOS and Android. It allows you to override any platform entity properties.
+
+### Preventing unnecessary ScreenView event for Android
+
+Previously a screen view event was tracked again by the screen view autotracking ADD LINK feature when the app moved to foreground. This is not expected because the screen doesn't change when the app is in background, and it is not consistent with how the screen view autotracking works on iOS. The extra event will no longer be tracked.
+
+### Event entities API for iOS
+
+To standardise the behaviour between trackers, we've changed how the `entities()` method of all events works on iOS. Previously, calling `event.entities(newListEntities)` replaced all the context entities currently attached to the event. Now, the new entities are appended instead.
+
+There's no change to the behaviour of the variable `entities`, so you could still replace them all using `event.entities = newListEntities`.
+
+## Non-optional tracker on iOS
+
+In the v5.x of the iOS tracker, `createTracker` returned an optional `TrackerController?`. This was an oversight. In v6, the iOS tracker again returns a non-optional `TrackerController`.
+
+## Concurrency on iOS
+
+The return type of the `Tracker.track()` method has changed from `UUID?` to `UUID`. Previously, `nil` could be returned if the tracker was paused, or if the event was filtered out and not actually tracked. From v6 onwards it will always return a UUID. If the event is tracked, this will be the `eventId`.
+
+This change was necessary as part of the comprehensive refactoring of the tracker thread model. The iOS tracker now uses a global dispatch queue. This single queue makes the tracker much safer. Almost all actions are now performed concurrently. Network requests in Emitter are still asynchronous.
+
+## Changes to the `EventStore` interface
+
+A new method, `removeOldEvents()` has been added to the `EventStore` protocol on both trackers. This method is used in the new feature that deletes events from storage if they get too old (by default, 30 days). Also, if too many events collect in the `EventStore`, the older ones will be deleted (default 1000 events).
+
+For the Android tracker, we have updated the `EventStore` interface to remove the optional types.
+
+## Non-breaking changes and bug fixes
```

Comment:
These are more like changelog items, do you think that's relevant here?

Reply:
I think its easier to have all the info about the release in one place, rather than having to check release notes for both trackers. If a user is coming here to find out about breaking changes they may as well find out about everything. Links to the releases on Github, for the actual release notes/changelog, would be good as well.

Reply:
Sure, that sounds good!